.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+/logs/orderApp-20230711.log
```

```suggestion
/logs/*
```
we should go for a more universal solution, let's ignore all the logs in the `logs` folder

---

Done